src/pages/AllCoins/AllCoins.css.js

```diff
@@ -0,0 +1,151 @@
+import styled from "styled-components";
+
+export const BulletPoint = styled.span`
+    margin-right: 6px;
+`;
+
+export const Cell = styled.div`
+ /* outline: 1px solid green; */
+ position: relative;
+ color: ${(props) => props.number ? (props.number < 0 ? "#FE1040" : "#00FC2A") : ''};

```

Comment:
if number is gonna be 0, then it' not gonna pass, zero is falsy

Reply:
Replaced with props.hasOwnProperty('number')

Reply:
Just changed it to !isNaN(props.number) since i'm now passing 'number' each time.